generator/cli/generate-type-script.ts

```diff
@@ -253,7 +253,9 @@ ${indentBlock(4, escapeBackticks(f.content))}
       type TemplateParameters = BaseTemplateParameters
 
       const templateParameterDefaults: Required<TemplateParameters> = {
-        datasourceProvider: Data.PrismaDatasourceProviderName.postgresql
+        datasourceProvider: Data.PrismaDatasourceProviderName.postgresql,
+        repositoryOwner: "undefined",
```

Comment:
Even though `BaseTemplateParameters` type should allow these fields to be optional https://github.com/prisma/templates-node/pull/24/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR29 
I get type errors at run time that these are not defined